api/follow/follow.spec.js

```diff
@@ -0,0 +1,68 @@
+const chai = require('chai');
+
+const should = chai.should();
+
+const app = require('../../app');
+
+const request = require('supertest');
+
+describe('/follow api', () => {
+  const follower = {
+    cid: 1,
+    mid: 2,
+  };
+
+  it('it should create a new Follower', (done) => {
+    request(app)
+      .post('/follow')
+      .send(follower)
+      .expect(201)
+      .end((err, res) => {
+        if (err) { done(err); return; }
+        res.body.cid.should.be.equal(follower.cid);
+        res.body.mid.should.be.equal(follower.mid);
+        done();
+      });
```

Comment:
Test case in not complete, because, we are only checking the response. However, we don't know whether the information was saved or not.